src/components/Dashboard/ProjectHeader.tsx

```diff
@@ -73,18 +73,60 @@ export default function ProjectHeader() {
         </div>
 
         <div style={{ flex: 1 }}>
-          <h1
+          <div
```

Comment:
Here, we're moving the `h1` and the project id under the same `div`. This way, we have more control over the project description layout